src/pages/_document.js

```diff
@@ -0,0 +1,34 @@
+import Document, { Head, Main, NextScript } from 'next/document'
+import { ServerStyleSheet } from 'styled-components'
+
+// Import base styles
+import '../styles/reset.css'
+import '../styles/style.css'
```

Comment:
We probably should alias `@styles`?

Reply:
These files will only be imported once, but why not... 🤷‍♂️